package.json

```diff
@@ -71,7 +71,8 @@
     "webpack-cli": "^2.0.12",
     "webpack-dev-server": "^2.2.0",
     "webpack-merge": "^4.1.2",
-    "webpack-strip": "^0.1.0"
+    "webpack-strip": "^0.1.0",
+    "lodash": "latest"
```

Comment:
Lets avoid latest tags and use the current latest as the version used.

Reply:
Also, I think it should be in dependencies if you're using it in the component.

Reply:
ok